src/Facility.ConformanceApi/GetWidgetRequestDto.g.cs

```diff
@@ -57,6 +57,9 @@ public override bool Validate(out string? errorMessage)
 			if (Id == null)
 				return ServiceDataUtility.GetRequiredFieldErrorMessage("id");
 
+			if (Id != null && Id < 0)
+				return ServiceDataUtility.GetInvalidFieldErrorMessage("Id", "Must be at least 0");
```

Comment:
```suggestion
				return ServiceDataUtility.GetInvalidFieldErrorMessage("id", "Must be at least 0.");
```

As on line 58 above, field names in error messages should use FSD casing.

Also, like other error messages, these should use sentence punctuation.